code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -152,5 +150,591 @@ datum/reagent/medicine/spaceacillin
+
+datum/reagent/medicine/omnizine/overdose_process(var/mob/living/M as mob)
+	M.adjustToxLoss(1.5*REM)
+	M.adjustOxyLoss(1.5*REM)
+	M.adjustBruteLoss(1.5*REM)
+	M.adjustFireLoss(1.5*REM)
+	..()
+	return
+
+datum/reagent/medicine/calomel
+	name = "Calomel"
+	id = "calomel"
+	description = "Quickly purges the body of all chemicals. If your health is above 20, toxin damage is dealt. When you hit 20 health or lower, the damage will cease."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+
+datum/reagent/medicine/calomel/on_mob_life(var/mob/living/M as mob)
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		if(R != src)
+			M.reagents.remove_reagent(R.id,2.5)
+	if(M.health > 20)
+		M.adjustToxLoss(2.5*REM)
+	..()
+	return
+
+datum/reagent/medicine/potass_iodide
+	name = "Potassium Iodide"
+	id = "potass_iodide"
+	description = "Reduces low radiation damage very effectively."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 2 * REAGENTS_METABOLISM
+
+datum/reagent/medicine/potass_iodide/on_mob_life(var/mob/living/M as mob)
+	if(M.radiation > 0)
+		M.radiation--
+	if(M.radiation < 0)
+		M.radiation = 0
+	..()
+	return
+
+datum/reagent/medicine/pen_acid
+	name = "Pentetic Acid"
+	id = "pen_acid"
+	description = "Reduces massive amounts of radiation and toxin damage while purging other chemicals from the body. Has a chance of dealing brute damage."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+
+datum/reagent/medicine/pen_acid/on_mob_life(var/mob/living/M as mob)
+	if(M.radiation > 0)
+		M.radiation -= 4
+	M.adjustToxLoss(-2*REM)
+	if(prob(33))
+		M.adjustBruteLoss(0.5*REM)
+	if(M.radiation < 0)
+		M.radiation = 0
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		if(R != src)
+			M.reagents.remove_reagent(R.id,2)
+	..()
+	return
+
+datum/reagent/medicine/sal_acid
+	name = "Salicyclic Acid"
+	id = "sal_acid"
+	description = "If you have less than 50 brute damage, it heals 0.25 unit. If overdosed it will deal 0.5 brute damage if the patient has less than 50 brute damage already."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	overdose_threshold = 25
+
+datum/reagent/medicine/sal_acid/on_mob_life(var/mob/living/M as mob)
+	if(M.getBruteLoss() < 50)
+		M.adjustBruteLoss(-0.25*REM)
+	..()
+	return
+
+datum/reagent/medicine/sal_acid/overdose_process(var/mob/living/M as mob)
+	if(M.getBruteLoss() < 50)
+		M.adjustBruteLoss(0.5*REM)
+	..()
+	return
+
+datum/reagent/medicine/salbutamol
+	name = "Salbutamol"
+	id = "salbutamol"
+	description = "Quickly heals oxygen damage while slowing down suffocation. Great for stabilizing critical patients!"
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+
+datum/reagent/medicine/salbutamol/on_mob_life(var/mob/living/M as mob)
+	M.adjustOxyLoss(-3*REM)
+	if(M.losebreath >= 4)
+		M.losebreath -= 2
```

Comment:
It was -4 and being called once every two life(). Now that it's called every life() the effect needs to be halved.